contracts/contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -254,7 +254,7 @@ namespace eosiosystem {
        */
       double              last_vote_weight = 0; /// the vote weight cast the last time the vote was updated
       time_point          vote_mature_time;
-      
+
```

Comment:
Maybe revert this changes? File included in PR only because of spaces seems weird